config/api_platform/playlist_slide.yaml

```diff
@@ -6,7 +6,7 @@ resources:
     itemOperations:
       get:
         # It needs to be here to generate id's in the collection lists.
-        path: '/playlist-slides/{id}'
+        path: '/playlistSlides/{id}'
```

Comment:
Our config is set to `api_platform.path_segment_name_generator.dash` https://api-platform.com/docs/core/operation-path-naming/#configuration